applications/OptimizationApplication/tests/test_material_properties_control.py

```diff
@@ -0,0 +1,111 @@
+
```

Comment:
put it under a folder dedicated to controls tests

Reply:
done :)